operations/configuration_management/Version_Control_Service_VCS.md

```diff
@@ -21,9 +21,9 @@ ncn# kubectl get secret -n services vcs-user-credentials \
 
 The initial VCS login credentials for the `crayvcs` user are stored in three places:
 
-- `vcs-user-credentials` Kubernetes secret: This is used to initialize the other two locations, as well as providing a place where other users can query for the password.
-- VCS \(Gitea\): These credentials are used when pushing to Git using the default username and password. The password should be changed through the Gitea UI.
-- Keycloak: These credentials are used to access the VCS UI. They must be changed through Keycloak.
+-   `vcs-user-credentials` Kubernetes secret: This is used to initialize the other two locations, as well as providing a place where other users can query for the password.
+-   VCS \(Gitea\): These credentials are used when pushing to Git using the default username and password. The password should be changed through the Gitea UI.
+-   Keycloak: These credentials are used to access the VCS UI. They must be changed through Keycloak.  For more information on accessing Keycloak, see [Access the Keycloak User Management UI](../security_and_authentication/Access_the_Keycloak_User_Management_UI.md)
```

Comment:
- Remove the extra space before the second sentence
- Add a period at the end of the second sentence

Reply:
fixed